public/app/plugins/panel/canvas/globalStyles.ts

```diff
@@ -3,276 +3,253 @@ import { css } from '@emotion/react';
 import { GrafanaTheme2 } from '@grafana/data';
 
 export function getGlobalStyles(theme: GrafanaTheme2) {
-  return css`
```

Comment:
This is the change I'm most worried about haha, but I think it should be ok.. but would appreciate a review of this (cc @adela-almasan 😬)